populator-machinery/controller.go

```diff
@@ -562,98 +720,101 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 		// Record start time for populator metric
 		c.metrics.operationStart(pvc.UID)
```

Comment:
Can be addressed as a followup, but it may be good to distinguish if we did the population using a pod or a function.

Reply:
Added a TODO comment for later improvement